config/printer-creality-cr6se-2020.cfg

```diff
@@ -0,0 +1,119 @@
+# This file contains pin mappings for the stock 2020 Creality CR6-SE.
+# To use this config, during "make menuconfig" select the
+# STM32F103 with a "28KiB bootloader" and with "Use USB for
+# communication" disabled.
+
+# To enable the stock probe, in "make menuconfig" select
+# "Enable extra low-level configuration options" and set "!PA5"
+# for "GPIO pins to set at micro-controller startup".
+
+# Flash this firmware by copying "out/klipper.bin" to a SD card and
+# turning on the printer with the card inserted. The firmware
+# filename must end in ".bin" and must not match the last filename
+# that was flashed.
+
+# See docs/Config_Reference.md for a description of parameters.
+
+[stepper_x]
+step_pin: PB8
+dir_pin: !PB7
+enable_pin: !PC3
+rotation_distance: 2.5
+microsteps: 1
+full_steps_per_rotation: 200
+endstop_pin: PC4
+position_min: -5
+position_endstop: -5
+position_max: 235
+homing_speed: 50
+
+[stepper_y]
+step_pin: PB6
+dir_pin: PB5
+enable_pin: !PC3
+rotation_distance: 2.5
+microsteps: 1
+full_steps_per_rotation: 200
+endstop_pin: PC5
+position_min: -2
+position_endstop: -2
+position_max: 235
+homing_speed: 50
+
+[stepper_z]
+step_pin: PB4
+dir_pin: !PB3
+enable_pin: !PC3
+rotation_distance: 0.5
+microsteps: 1
+full_steps_per_rotation: 200
+endstop_pin: probe:z_virtual_endstop
+position_min: -1.5
+position_max: 250
+homing_speed: 4
+second_homing_speed: 1
+homing_retract_dist: 2.0
+
+[extruder]
+max_extrude_only_distance: 1000.0
+step_pin: PC2
+dir_pin: !PB9
+enable_pin: !PC3
+rotation_distance: 1.9048
+microsteps: 1
+full_steps_per_rotation: 200
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+pressure_advance: 0.195
```

Comment:
It's preferable to not specify pressure_advance in any of the example config files, because there is no default generally applicable default value - it changes with each spool of filament.

Reply:
I agree, I just forgot to remove it.